kvmagent/kvmagent/test/test_parsing_multipath.py

```
@@ -1,81 +1,66 @@
 
-    assert write_multipath_conf('multipath.conf', blacklist)
-    assert not write_multipath_conf('multipath.conf')
+    assert multipath.write_multipath_conf('multipath.conf', blacklist)
+    assert not multipath.write_multipath_conf('multipath.conf')
     with open("multipath.conf",
          "r") as fd:
         text1 = multipath.parse_multipath_conf(fd)
     with open("multipath.conf.out", "r") as fd1:
         text2 = multipath.parse_multipath_conf(fd1)
-    assert text1 == text2
+    assert cmp(text1, text2) == 0
```

Ruff (0.13.1)

Undefined name `cmp`

(F821)